packages/connect-node/tsconfig.build.json

```diff
@@ -1,5 +1,9 @@
 {
   "files": ["src/index.ts"],
   "include": ["src/**/*.spec.ts", "src/testdata/*.ts"],
-  "extends": "../../tsconfig.base.json"
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
```

Comment:
`connect-node` is the only package that causes problems on `Node16` module resolution. This is because of the `Long` library which is a transitive dependency through grpc-js.  It's kind of frustrating that one library brought in transitively and that further is only used for extra testing is inhibiting us here. Any thoughts? We could maybe run these extra tests on their own and create a `tsconfig.extra.json` or something. Then we could exclude them in `tsconfig.json` or `tsconfig.build.json`.

Reply:
Forgot to mention that we could use this workaround in the meantime also? 😬 

https://github.com/dcodeIO/long.js/issues/125#issuecomment-1897638682

Reply:
I'm okay with deleting packages/connect-node/src/extra. It's rotting here anyway. Maybe we'll make this available as a separate plugin or library. It's easy enough to pick it up from the history.